### PR TITLE
[7.x] Bootstrap 5 + PHP 8 + vanilla javascript

### PR DESCRIPTION
**Pull request by f3l1x**

### CODE REVIEW

**Comment by petrparolek** [comment]
bude i zpětná kompatibilita s BS 4?

**Comment by f3l1x** [comment]
Rekl bych, ze na to nemame kapacity ani prostredky. Nejspis ne.

**Comment by radimvaculik** [comment]
@f3l1x Přibalil bych do toho PR i bump `bootstrap-select`, protože to jde ruku v ruce.

File: src/templates/datagrid.latte [hunk 1]
```diff
@@ -20,7 +20,7 @@
 		{form filter, class => 'ajax'}
 			{if $control->hasOuterFilterRendering()}
 				{block outer-filters}
-					<div class="row text-right datagrid-collapse-filters-button-row" n:if="$control->hasCollapsibleOuterFilters()">
+					<div class="row text-end datagrid-collapse-filters-button-row" n:if="$control->hasCollapsibleOuterFilters()">
 						<div class="col-sm-12">
 							<button class="btn btn-xs btn-primary" type="button" data-toggle="collapse" data-target="#datagrid-{$control->getFullName()}-row-filters">
```
Comment (radimvaculik) [inline src/templates/datagrid.latte:25]:
```suggestion
							<button class="btn btn-xs btn-primary" type="button" data-bs-toggle="collapse" data-bs-target="#datagrid-{$control->getFullName()}-row-filters">
```

File: assets/datagrid.css [hunk 2]
```diff
@@ -26,6 +26,18 @@
...
+	text-decoration: none;
+}
+
+.datagrid a:hover {
+	text-decoration: underline;
+}
+
+.datagrid a.btn:hover {
+	text-decoration: none;
+}
```
Comment (radimvaculik) [inline assets/datagrid.css:39]:
```suggestion
.datagrid a:not(.btn):not(.dropdown-item):hover {
	text-decoration: underline;
}
```

**Comment by dakorpar** [comment]
@f3l1x @paveljanda What's the status of this? Any ETA? Can I help?

**Comment by radimvaculik** [comment]
It's ready. If you can test it, please do it.

**Comment by radimvaculik** [comment]
@dakorpar Rebased. Could you try it, please?

**Comment by radimvaculik** [comment]
@dakorpar  All good?

**Comment by foxycode** [comment]
Any progress with this? I tested using `"ublaboo/datagrid": "dev-master#554662a"` in `composer.json`. I see `select`s are deformed and filter hide button doesn't work either.

image

**Comment by radimvaculik** [comment]
@foxycode How do you load CSS?

**Comment by foxycode** [comment]
@radimvaculik That would be the problem, using CDN :slightly_smiling_face: 

Can you advise how to use CSS from this commit?

**Comment by radimvaculik** [comment]
Use webpack, gulp or another asset builder.

**Comment by foxycode** [comment]
@radimvaculik Tested, looks good.

**Comment by foxycode** [comment]
Any news about merge date? I am using it without problem for some time.

**Comment by radimvaculik** [comment]
@foxycode See https://github.com/contributte/datagrid/issues/1025

**Pull request merged**